infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -141,6 +142,21 @@ public LogUnitServer(ServerContext serverContext) {
         logCleaner = new StreamLogCompaction(streamLog, 10, 45, TimeUnit.MINUTES, ServerContext.SHUTDOWN_TIMER);
     }
 
+    public LogUnitServer(ServerContext serverContext, StreamLog streamLog){
```

Comment:
Duplicate code in the 2 constructors. Please coalesce them if possible.

Reply:
Sounds good